packages/testing/src/test/Helpers.test.ts

```diff
@@ -0,0 +1,103 @@
+      const backendInitializationSpy = sinon.spy(PresentationBackend, "initialize");
+      const frontendInitializationSpy = sinon.spy(PresentationFrontend, "initialize");
+
+      await initialize();
+      expect(backendInitializationSpy.calledOnce);
+      expect(frontendInitializationSpy.calledOnce);
+
+      sinon.resetHistory();
+
+      await initialize();
+      expect(backendInitializationSpy.notCalled);
+      expect(frontendInitializationSpy.notCalled);
+    });
+
+    it("initializes PresentationBackend and PresentationFrontend with provided props", async () => {
+      const backendInitializationSpy = sinon.spy(PresentationBackend, "initialize");
+      const frontendAppStartupSpy = sinon.spy(NoRenderApp, "startup");
+      const props: PresentationTestingInitProps = { backendProps: { id: Guid.createValue() }, frontendApp: NoRenderApp };
+
+      await initialize(props);
+      expect(backendInitializationSpy.calledOnceWith(props.backendProps));
+      expect(frontendAppStartupSpy.calledOnce);
+    });
+  });
+
+  describe("terminate", () => {
+    it("terminates PresentationBackend and PresentationFrontend on terminate", async () => {
+      const backendTerminationSpy = sinon.spy(PresentationBackend, "terminate");
+      const frontendTerminationSpy = sinon.spy(PresentationFrontend, "terminate");
+      await initialize();
+
+      await terminate();
+      expect(backendTerminationSpy.calledOnce);
+      expect(frontendTerminationSpy.calledOnce);
+    });
+
+    it("does not terminate PresentationBackend and PresentationFrontend on second call to terminate", async () => {
+      const backendTerminationSpy = sinon.spy(PresentationBackend, "terminate");
+      const frontendTerminationSpy = sinon.spy(PresentationFrontend, "terminate");
+      await initialize();
+
+      await terminate();
+      expect(backendTerminationSpy.calledOnce);
+      expect(frontendTerminationSpy.calledOnce);
+
+      sinon.resetHistory();
+
+      await terminate();
+      expect(backendTerminationSpy.notCalled);
+      expect(frontendTerminationSpy.notCalled);
+    });
+
+    it("calls rimraf sync when initialized with DiskHierarchyCacheConfig", async () => {
+      const syncStub = sinon.stub(rimraf, "sync");
+      const props: PresentationTestingInitProps = { backendProps: { caching: { hierarchies: { mode: HierarchyCacheMode.Disk, directory: process.cwd() } } } };
+      await initialize(props);
+
+      await terminate();
+      expect(syncStub.calledOnce);
+    });
+
+    it("calls rimraf sync when initialized with HybridCacheConfig", async () => {
+      const syncStub = sinon.stub(rimraf, "sync");
+      const props: PresentationTestingInitProps = {
+        backendProps: { caching: { hierarchies: { mode: HierarchyCacheMode.Hybrid, disk: { mode: HierarchyCacheMode.Disk, directory: process.cwd() } } } },
+      };
+      await initialize(props);
+
+      await terminate();
+      expect(syncStub.calledOnce);
+    });
```

Comment:
Using `rimraf.sync` is an implementation detail of the goal to simply clear the cache directory. I suggest we rename the tests to describe the goal rather than this implementation detail, e.g. "clears cache directory when initialized with DiskHierarchyCacheConfig". 

Also, I don't think we need to pass a real (`process.cwd()`) directory here - simply `"/test/directory/"` should be sufficient. And we should assert that `rimraf.sync` is called with that directory.